functions/gravityForms/getGfFieldValidationSchema.js

```diff
@@ -1,39 +1,116 @@
 import getGfFieldId from '@/functions/gravityForms/getGfFieldId'
 import * as Yup from 'yup'
 
-// TODO Refactor this file for scaling purposes.
-
 /**
- * Map props to validation schemas.
+ * Create Yup string schema object.
  *
- * @param {Object} fieldData GravityForm field props.
+ * @author Mike England <mike.england@webdevstudios.com>
+ * @since 2021-01-07
```

Comment:
Please remove the `@since` and change `@link` to `@see`. `@author` should be WebDevStudios. 

We need to consistently follow valid JSDoc block tags across this entire project too. https://jsdoc.app/index.html#block-tags